### DISCUSSION TITLE
Deal with dot in column names

### DISCUSSION THREAD

**Discussion by qqlearn123** [duckdb/duckdb#7952] [Q&A] [answered]
Is there anyway to deal with dot in column names? I tried using either backtick (`) or square bracket ([) and they do not seem to work.

I am mostly using `duckdb` on `python` side.

**Comment by l1t1** [comment]
```
>>> import duckdb
>>> duckdb.sql('create table t ("a.b" int)')
>>> duckdb.sql('insert into t("a.b") values(100)')
>>> duckdb.sql('select "a.b" from t')
┌───────┐
│  a.b  │
│ int32 │
├───────┤
│   100 │
└───────┘
```

**Reply by qqlearn123** [reply]
Thanks. How about if I want to do something like t.a.b? It seems like t.'a.b' results into syntax error.

**Comment by l1t1** [comment]
select t."a.b" from t

**Comment by mycaule** [comment, marked as answer]
> ```
> >>> import duckdb
> >>> duckdb.sql('create table t ("a.b" int)')
> >>> duckdb.sql('insert into t("a.b") values(100)')
> >>> duckdb.sql('select "a.b" from t')
> ┌───────┐
> │  a.b  │
> │ int32 │
> ├───────┤
> │   100 │
> └───────┘
> ```

comment was ok you are supposed to use double quotes to escape the dots
